docker-compose.yml

```diff
@@ -2,12 +2,12 @@
 #dockstore can run  without elasticsearch but the search page requires elasticsearch to be running
 #kibana can be used alongside elasticsearch as a visual interface for the indices and send queries directly
 #this only brings up the postgres container, not the actual database
+#For detailed instructions on how to set up dockstore locally, refer to:
+#https://wiki.oicr.on.ca/display/DOC/Setting+Up+Dockstore+Locally#SettingUpDockstoreLocally-SettingupElasticsearch
```

Comment:
This is a public repo, but the link here has restricted access. I think you should remove these 2 lines.